vitrina/messages/helpers.py

```diff
@@ -7,22 +7,38 @@
 def prepare_email_by_identifier_for_sub(email_identifier, base_template_content,
```

Comment:
Do we still need this function? If not, it should be removed.

Reply:
I noticed that people are using this and another function in new functionalities, so I think this function should be left until this task will be merged to devel.